javascript-light-bulb/main.js

```diff
@@ -0,0 +1,18 @@
+var totalClick = 0;
+
+var light = document.querySelector('.circle');
+var background = document.querySelector('.container');
+
+function onAndOff(event) {
+  totalClick++;
+  if (totalClick % 2 !== 0) {
```

Comment:
You aren't using the click count here for anything other than checking if the value is even or odd. In this case, a variable holding a boolean value would be a better way of keeping track of if the light is currently on or not.